<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="255,1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_chart">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_operations">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_operations">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonStart">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReset">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_information">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Design th. (nm)::</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditDesignTh</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditDesignTh">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Current th. (nm):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditCurrentTh</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEditCurrentTh">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Value f_av:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditValueF_Av</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLineEdit" name="lineEditValueF_Av">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Remaining th. (nm):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditRemTh</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLineEdit" name="lineEditRemTh">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Active layer:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditActiveLayer</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditActiveLayer">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelRate">
            <property name="text">
             <string>Rate (A/s):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditRate</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditRate">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Execution t.:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditExecutionT</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="lineEditExecutionT">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Material:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditMaterial</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QLineEdit" name="lineEditMaterial">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelTotalLayers">
            <property name="text">
             <string>Total layers:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditTotalLayers</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditTotalLayers">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Remaining time (s):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditRemTime</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEditRemTime">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImportSubstrate"/>
     <addaction name="actionImportMaterialL"/>
     <addaction name="actionImportMaterialH"/>
     <addaction name="actionImportDesign"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="menuImport"/>
    <addaction name="actionProjects"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDesign">
    <property name="title">
     <string>&amp;Design</string>
    </property>
    <addaction name="actionDesign"/>
    <addaction name="actionLayerMaterial"/>
    <addaction name="actionSubstrate"/>
   </widget>
   <widget class="QMenu" name="menuD_eposition_settings">
    <property name="title">
     <string>D&amp;eposition settings</string>
    </property>
    <addaction name="actionProcessParameters"/>
    <addaction name="actionMainLog"/>
    <addaction name="actionVacuumMachineSettings"/>
    <addaction name="actionErrorPrevention"/>
   </widget>
   <widget class="QMenu" name="menu_Process_data">
    <property name="title">
     <string>&amp;Process data</string>
    </property>
    <addaction name="actionDataFitting"/>
    <addaction name="actionIterationExecutionTime"/>
    <addaction name="actionHistory"/>
   </widget>
   <widget class="QMenu" name="menu_Spectrometer">
    <property name="title">
     <string>&amp;Spectrometer</string>
    </property>
    <addaction name="actionSpectrometerSettings"/>
    <addaction name="actionDataCorrection"/>
    <addaction name="actionRawData"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionSimulationParameters"/>
    <addaction name="actionReverseEngineering"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDesign"/>
   <addaction name="menuD_eposition_settings"/>
   <addaction name="menu_Process_data"/>
   <addaction name="menu_Spectrometer"/>
   <addaction name="menu_Options"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProjects"/>
   <addaction name="separator"/>
   <addaction name="actionSubstrate"/>
   <addaction name="actionLayerMaterial"/>
   <addaction name="actionDesign"/>
   <addaction name="separator"/>
   <addaction name="actionProcessParameters"/>
   <addaction name="actionDataFitting"/>
   <addaction name="actionHistory"/>
   <addaction name="separator"/>
   <addaction name="actionIterationExecutionTime"/>
   <addaction name="separator"/>
   <addaction name="actionSaveSpectra"/>
  </widget>
  <action name="actionProjects">
   <property name="text">
    <string>&amp;Projects...</string>
   </property>
  </action>
  <action name="actionSubstrate">
   <property name="text">
    <string>&amp;Substrate...</string>
   </property>
  </action>
  <action name="actionLayerMaterial">
   <property name="text">
    <string>&amp;Layer Material...</string>
   </property>
  </action>
  <action name="actionDesign">
   <property name="text">
    <string>&amp;Design...</string>
   </property>
  </action>
  <action name="actionProcessParameters">
   <property name="text">
    <string>&amp;Process Parameters...</string>
   </property>
  </action>
  <action name="actionDataFitting">
   <property name="text">
    <string>&amp;Data Fitting...</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>&amp;History...</string>
   </property>
  </action>
  <action name="actionIterationExecutionTime">
   <property name="text">
    <string>&amp;Iteration Execution Time...</string>
   </property>
  </action>
  <action name="actionSaveSpectra">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Spectra</string>
   </property>
  </action>
  <action name="actionSimulationParameters">
   <property name="text">
    <string>&amp;Simulation Parameters...</string>
   </property>
  </action>
  <action name="actionReverseEngineering">
   <property name="text">
    <string>&amp;Reverse Engineering...</string>
   </property>
  </action>
  <action name="actionSpectrometerSettings">
   <property name="text">
    <string>&amp;Spectrometer Settings...</string>
   </property>
  </action>
  <action name="actionDataCorrection">
   <property name="text">
    <string>&amp;Data Correction...</string>
   </property>
  </action>
  <action name="actionRawData">
   <property name="text">
    <string>&amp;Raw Data...</string>
   </property>
  </action>
  <action name="actionMainLog">
   <property name="text">
    <string>&amp;Main Log...</string>
   </property>
  </action>
  <action name="actionVacuumMachineSettings">
   <property name="text">
    <string>&amp;Vacuum Machine Settings...</string>
   </property>
  </action>
  <action name="actionErrorPrevention">
   <property name="text">
    <string>&amp;Error Prevention...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionImportSubstrate">
   <property name="text">
    <string>Substrate...</string>
   </property>
  </action>
  <action name="actionImportMaterialL">
   <property name="text">
    <string>Material L...</string>
   </property>
  </action>
  <action name="actionImportMaterialH">
   <property name="text">
    <string>Material H...</string>
   </property>
  </action>
  <action name="actionImportDesign">
   <property name="text">
    <string>Design...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
